documentation/onboard-service.md

```diff
@@ -109,9 +111,14 @@ Take note of the name servers of the newly created DNS zone and request delegati
 - [Infrastructure and networks](https://technical-guidance.education.gov.uk/infrastructure/hosting/dns/#service-now) for the education.gov.uk zone
 - [GDS](https://technical-guidance.education.gov.uk/infrastructure/hosting/dns/#official-service-gov-uk-domains) for the service.gov.uk zone
 
+### Create domain.sh in global config
```

Comment:
domain.sh should already be there as it was created by the template: https://github.com/DFE-Digital/teacher-services-cloud/blob/main/templates/new_service/global_config/domains.sh

Reply:
Same as above. Should I remove or use something like "Ensure ..."?